tox.ini

```diff
@@ -0,0 +1,94 @@
+; a generative tox configuration, see: https://tox.readthedocs.io/en/latest/config.html#generative-envlist
+
+[tox]
+envlist =
+    clean,
+    docs,
+    {py35,py36,py37,pypy,pypy3},
+    flake8,
+    report
+
+[testenv]
+basepython =
+    pypy: {env:TOXPYTHON:pypy}
+    pypy3: {env:TOXPYTHON:pypy3}
+    py35: {env:TOXPYTHON:python3.5}
+    {py36,docs,spell}: {env:TOXPYTHON:python3.6}
+    py37: {env:TOXPYTHON:python3.7}
+    {clean,check,report,codecov}: {env:TOXPYTHON:python3}
+setenv =
+    PYTHONPATH={toxinidir}/tests
+    PYTHONUNBUFFERED=yes
+passenv =
+    *
+usedevelop = false
+deps =
+    astropy>=3.1,<4.*
+    astroquery>=0.3.9
+    jplephem
+    matplotlib>=2.0,!=3.0.1
+    numba>=0.39
+    numpy
+    pandas
+    plotly>=3.6,<4.*
+    scipy>=1.0
+commands =
+    {posargs:pytest --cov --cov-report=term-missing -vv src/poliastro/tests}
+
+[testenv:check]
+deps =
+    black
+    docutils
+    isort
+    flake8
+    mypy
+    pygments
+skip_install = true
+usedevelop = false
+commands =
+    python setup.py check --strict --metadata --restructuredtext
+    flake8 src setup.py
+    isort --check-only --diff --recursive --project poliastro --section-default THIRDPARTY src setup.py
+    black --check src setup.py
+    mypy --ignore-missing-imports --check-untyped-defs --no-strict-optional src
+
+[testenv:docs]
+description = invoke sphinx-build to build the HTML docs
+extras = docs
+commands = sphinx-build -d "{toxworkdir}/docs_doctree" docs/source "{toxworkdir}/docs_out" --color -W -bhtml
+
+[testenv:codecov]
+deps =
+    codecov
+skip_install = true
+commands =
+    coverage xml --ignore-errors
+    codecov []
+
+[flake8]
```

Comment:
It's a bit strange to me to put the `flake8` config in `tox.ini`. Does it work if it's invoked _without_ tox as well? Maybe we can leave it at `setup.cfg`?

Reply:
Sure! Moving to setup.cfg :+1: